jlib/new.go

```diff
@@ -259,23 +319,32 @@ func ObjectsToDocument(input interface{}) (interface{}, error) {
 		return nil, errors.New("$objectsToDocument input must be an array of objects")
 	}
 
-	output := make(map[string]interface{}) // Initialize the output map
-	// Iterate through each item in the input
+	output := make(map[string]interface{})
 	for _, itemToInterface := range trueInput {
 		item, ok := itemToInterface.(map[string]interface{})
 		if !ok {
-			return nil, errors.New("$objectsToDocument input must be an array of objects with Code and Value fields")
+			return nil, errors.New("$objectsToDocument input must be an array of objects with Code and Val/Value fields")
 		}
-		// Call setValue for each item to set the value in the output map
+
 		code, ok := item["Code"].(string)
 		if !ok {
```

Comment:
should this return an error? If it's missing code, could be a user error.
Or is that expected behaviour, if yes, you can ignore this comment.

Reply:
@JamesXiatech - I'm not sure on the behaviour, could you confirm if missing `Code` is good to disregard this line and drop it, or if you need it to throw an error so it can be correct?

Reply:
Confirmed with @JamesXiatech that the desired behaviour is to error. Code has been updated to replace the `continue` with an Error.